gpbft/gpbft.go

```diff
@@ -86,22 +100,24 @@ type Payload struct {
 	Round uint64
 	// GossiPBFT step name.
 	Step Phase
-	// Chain of tipsets proposed/voted for finalisation.
-	// Always non-empty; the first entry is the base tipset finalised in the previous instance.
-	Value ECChain
+	// The common data.
+	Data InstanceData
```

Comment:
Can we work a bit on this naming? Perhaps AccessoryData or BaseData? It all references things that are already finalised by a prior instance, right?

Then rename InstanceData similarly (it's not data about this instance, it's about prior instances).

Reply:
By consensus: supplemental data.